helm/polaris/values.yaml

```
@@ -67,18 +67,21 @@ revisionHistoryLimit: ~
 
 # -- Security context for the polaris pod. See https://kubernetes.io/docs/tasks/configure-pod-container/security-context/.
 podSecurityContext:
-  {}
-  # fsGroup: 2000
+  # -- GID 10001 is compatible with Polaris OSS default images; change this if you are using a different image.
```

```suggestion
  # -- GID 10001 is for the Polaris OSS default images; only change this if you are using a different image.
```
(and similar for the UID below)
WDYT?